### ISSUE TITLE
Broken build on macOS

### ISSUE THREAD

**Issue by Yicong-Huang** [google/googletest#3059]
After new commits on 10/14/2020, now building on macOS 11.0 will give the following error:

```In file included from ../cmake-build-debug/googletest-prefix/src/googletest/googlemock/src/gmock-all.cc:39:
In file included from ../cmake-build-debug/googletest-prefix/src/googletest/googlemock/include/gmock/gmock.h:61:
In file included from ../cmake-build-debug/googletest-prefix/src/googletest/googlemock/include/gmock/gmock-function-mocker.h:42:
In file included from ../cmake-build-debug/googletest-prefix/src/googletest/googlemock/include/gmock/gmock-spec-builders.h:75:
../cmake-build-debug/googletest-prefix/src/googletest/googlemock/include/gmock/gmock-matchers.h:2983:31: error: unused parameter 'a' [-Werror,-Wunused-parameter]
void VariadicExpand(const T (&a)[N]) {}
                              ^
../cmake-build-debug/googletest-prefix/src/googletest/googlemock/include/gmock/gmock-matchers.h:3043:38: error: implicit conversion changes signedness: 'int' to 'std::__1::vector<testing::StringMatchResultListener, std::__1::allocator<testing::StringMatchResultListener> >::size_type' (aka 'unsigned long') [-Werror,-Wsign-conversion]
      PrintIfNotEmpty(inner_listener[failed_pos].str(), listener->stream());

                      ~~~~~~~~~~~~~~ ^~~~~~~~~~
```

Apple clang version 12.0.0 (clang-1200.0.32.4)
Target: x86_64-apple-darwin20.1.0
Thread model: posix
InstalledDir: /Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin

**Comment by skypjack** [comment]
Confirmed also with clang on linux.

**Comment by derekmauro** [comment]
Should be fixed by 3c95bf552405fd0cc63cea0ca2f6c4cd89c8d356.
